component/scripts/approve.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ONE_WEEK_AGO=$(date -v-${APPROVE_DELAY} -u +%Y-%m-%dT%H:%M:%SZ)
+INSTALLPLAN=$(kubectl -n syn-airlock-microgateway get ip -ojson | jq -r --arg date ${ONE_WEEK_AGO} '.items | sort_by(.metadata.creationTimestamp) | reverse | [.[] |  select((.spec.approved != true) and (.metadata.creationTimestamp < $date))][0] | .metadata.name')
```

Comment:
The `kubectl get` in this command should inherit the namespace configured in the component!

```suggestion
INSTALLPLAN=$(kubectl -n "${NAMESPACE}" get ip -ojson | jq -r --arg date "${DELAY_TIMESTAMP}" '.items | sort_by(.metadata.creationTimestamp) | reverse | [.[] |  select((.spec.approved != true) and (.metadata.creationTimestamp < $date))][0] | .metadata.name')
```